### ISSUE TITLE
Using io_uring in Linux?

### ISSUE THREAD

**Issue by byte-sourcerer** [VictoriaMetrics/VictoriaMetrics#6667]
### Is your question request related to a specific component?

Does VictoriaMetrics employ the io_uring functionality in Linux? Could its integration potentially enhance the performance speed of vmstorage?

### Describe the question in detail

N/A

### Troubleshooting docs

- [ ] General - https://docs.victoriametrics.com/troubleshooting/
- [ ] vmagent - https://docs.victoriametrics.com/vmagent/#troubleshooting
- [ ] vmalert - https://docs.victoriametrics.com/vmalert/#troubleshooting

**Comment by jiekun** [comment]
Hello thanks for the question. I'm not an expert of this but did a quick search in the code base and it seems no relevant result of `io_uring`. Also it might be part of the Go runtime so I'm not sure how it could be used or integrated into VictoriaMetrics.

Would you mind providing more info, like how could we enable this feature and I'm more than happy to discuss and do a test.

**Comment by tiny-pangolin** [comment]
I do not believe victoriametrics implements io_uring at this time since io_uring only works on Linux with kernel version 5.6 or later. I believe using io_uring would require dropping certain platforms or adding additional logic to see if io_uring is available and have separate code paths for io_uring and no io_uring
